.circleci/config.yml

```diff
@@ -86,6 +86,9 @@ jobs:
       - checkout
       - attach_workspace:
           at: workspace
+      - run:
+          name: install from constraints file
+          command: pip install -r constraints.txt
```

Comment:
Why do you need to install constraints on the manager test? It uses the wagon, not a local env.

Reply:
I have updated to install wagon to system for resolve dependencies.